typescript/infra/test/core.test.ts

```diff
@@ -91,7 +89,9 @@ describe('core', async () => {
     it('can be resumed from partial contracts', async () => {
       sinon.restore(); // restore normal deployer behavior
 
+      //@ts-ignore operand not optional, ignore for this test
```

Comment:
The fixing of some types in this file caused these to be required some deployedContracts is no longer `any`